promscale/installation/docker.md

```diff
@@ -17,21 +16,15 @@ packages and instructions, see the
 ### Installing Promscale with Docker
 1.  Use Docker to create a network for Promscale and TimescaleDB:
     ```bash
-    docker network create --driver bridge promscale-timescaledb
+    docker network create --driver bridge promscale
     ```
-1.  Install TimescaleDB in a Docker container, and make sure the service is
-    running. For instructions, see
-    [the TimescaleDB Docker installation section][tsdb-docker]. Promscale stores
-    all metrics data scraped from Prometheus targets in this TimescaleDB service.
-1.  Run the Promscale Docker container. This example creates a TimescaleDB
-    instance called `timescaledb`, on a network named `promscale-timescaledb`. The
-    container runs in the background, and the container ID is shown after it is
-    created. It also port forwards to port `5432` on your local system:
+1.  Install TimescaleDB in a Docker container on a network named `promscale`. It also port forwards to port `5432` on your local system:
     ```bash
-    docker run --name timescaledb \
-        --network promscale-timescaledb \
-        -e POSTGRES_PASSWORD=<password> -d -p 5432:5432 \
-        timescaledev/timescaledb-ha:pg12-latest
+    docker run --name timescaledb -e POSTGRES_PASSWORD=<password> -d -p 5432:5432 --network promscale timescaledev/promscale-extension:latest-ts2-pg13 postgres -csynchronous_commit=off
```

Comment:
Using <REPLACEABLE_TEXT> is standard in tech docs. The idea being that when people copy and paste, the command will fail until they have replaced it. I think allowing people to copy paste without changing things that need changing is inadvisable.

Reply:
@VineethReddy02 Can you comment on where `synchronous_commit=off` comes from? Is that a recommended setting?

Could we also line-break this command so that it's completely visible on the docs page?

Reply:
I agree with both of you :) Some docs have a button to auto-generate a password so the user can just copy and paste so the password is random and secure and they know it was auto-generated because they had to take action.

Reply:
So I'd say for now we leave as is.

Reply:
AFAIK synchronous_commit=off is a recommended setting and we use it in Timescale Cloud for example.

Reply:
We've broken up one line (the Promscale one) but not the other. PR added: https://github.com/timescale/docs/pull/684